third_party/pyth/pyth_publisher.py

```diff
@@ -164,6 +164,9 @@ def add_symbol(num: int):
                 next_new_symbol_id += 1
                 dynamically_added_symbols += 1
 
+        if dynamically_added_symbols >= PYTH_DYNAMIC_SYMBOL_COUNT:
```

Comment:
I generally like the permissive comparison, but I think the `>` case would be a bug in this script, maybe we could yell about it and only be quiet for `==`?